CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Page navigation is now dynamically rendered reducing file sizes.
```

Comment:
Please add issue number so that we can more easily get what we were referring to. If there is no issue, you might instead add PR number. See any other scraper (youtube, ted, ...).